bo/zaehlzeitdefinition.go

```diff
@@ -0,0 +1,28 @@
+package bo
+
+import (
+	"github.com/hochfrequenz/go-bo4e/com"
+	"github.com/hochfrequenz/go-bo4e/enum/definitionennotwendigkeit"
+	"time"
+)
+
+// / <summary>
+// / Der NB bzw. LF nutzt Zählzeitdefinitionen für die Tarifierung von Werten.
+// / <remarks>Zaehlzeitdefinitionen werden in der Marktkommunikation mit Prüfidentifikator 25001 (UTILTS) übermittelt</remarks>
+// / Eine Zählzeitdefinition umfasst dabei eine Liste von möglichen Zählzeiten,
+// / den dazugehörigen Registern und der tatsächlich ausgerollten Zählzeit (wenn diese elektronisch übermittelt wird)
+// / </summary>
+type Zaehlzeitdefinition struct {
+	Geschaeftsobjekt
+	Beginndatum            *time.Time                                           `json:"beginndatum,omitempty"`
+	Endedatum              *time.Time                                           `json:"endedatum,omitempty"`
+	Version                time.Time                                            `json:"version,omitempty"`
+	Notwendigkeit          *definitionennotwendigkeit.DefinitionenNotwendigkeit `json:"notwendigkeit,omitempty"`
+	Zaehlzeiten            []com.Zaehlzeit                                      `json:"zaehlzeiten,omitempty"`
+	Zaehlzeitregister      []com.Zaehlzeitregister                              `json:"zaehlzeitregister,omitempty"`
+	AusgerollteZaehlzeiten []com.AusgerollteZaehlzeit                           `json:"ausgerolltezaehlzeiten,omitempty"`
```

Comment:
```suggestion
	AusgerollteZaehlzeiten []com.AusgerollteZaehlzeit                           `json:"ausgerollteZaehlzeiten,omitempty"`
```
https://github.com/Hochfrequenz/BO4E-dotnet/blob/c888824718356f5c125ad72763ec509ceb6334ca/BO4E/BO/Zaehlzeitdefinition.cs#L119